imap_processing/codice/decompress.py

```diff
@@ -74,13 +74,13 @@ def _apply_lossy_b(compressed_value: int) -> int:
     return LOSSY_B_TABLE[compressed_value]
 
 
-def _apply_lzma_lossless(compressed_value: int) -> int:
+def _apply_lzma_lossless(compressed_value: bytes) -> int:
```

Comment:
Looking at the unit tests, it looks like this should allow both `bytes` and `int`

Reply:
@sdhoyt 
I had to switch around types for this one because the `lama.decompress()` only takes objects that are `bytes`. If the test cases take `int`'s then to satisfy mypy the int would need to be converted to a byte then passed to the decompress function. 
Should I add an if-statement to change `int` to `byte` if it is passed in? That way the code honors the tests

Reply:
I might be missing something, but it looks like decompress expects type `int` for `compressed_value`:

```python
decompress(compressed_value: int, algorithm: IntEnum) -> int:
```

Reply:
Oh, I Understand where the confusion is. `decompress()` is different from `lzma.decompress()`( the previous message got autocorrected). From what I can tell everything expects an `int` to be passed in and passed out. However in `_apply_lzma_lossless()` the `lzma.decompress()` is a function from the lzma module and needs a `bytes` type.

Reply:
I went ahead and pushed what I feel it all should look like. Let me know what you think!

Reply:
Ah gotcha. Sounds good